iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -805,7 +841,10 @@ + (void)downloadIOSParamsWithAppId:(NSString *)appId {
         if (result[IOS_RECEIVE_RECEIPTS_ENABLE] != (id)[NSNull null])
             [OneSignalUserDefaults.initShared saveBoolForKey:OSUD_RECEIVE_RECEIPTS_ENABLED withValue:[result[IOS_RECEIVE_RECEIPTS_ENABLE] boolValue]];
 
-        [OSOutcomesUtils saveOutcomeParamsForApp:result];
+        if (result[IOS_OUTCOMES_V2_SERVICE_ENABLE])
```

Comment:
This should be under `outcomes`, like we have for Android

Reply:
good catch!